Examples/Python/python/acts/examples/simulation.py

```diff
@@ -555,10 +553,12 @@ def addGeant4(
     else:
         particles_selected = "particles_input"
 
-    g4detector = DDG4DetectorConstruction(geometryService)
+    if g4detectorConstruction is None:
```

Comment:
Hmm is this redundant? There is no default for `g4detectorConstruction`, so someone would need to pass explicitly `None` for the `g4detectorConstruction` to the function, which is not very likely.

Reply:
you are right - I will make use of this `if` here https://github.com/acts-project/acts/blob/83298cbe8bc08761113c4b132f43acd89c9b5edf/Examples/Python/python/acts/examples/simulation.py#L589-L593 in https://github.com/acts-project/acts/pull/1638

so I think its fine to merge this temporarily even tho is has no effect